pkg/operator/ceph/object/spec.go

```diff
@@ -29,7 +29,7 @@ import (
 	"k8s.io/apimachinery/pkg/util/intstr"
 )
 
-func (c *clusterConfig) startDeployment(rgwConfig *rgwConfig) (*apps.Deployment, error) {
+func (c *clusterConfig) startDeployment(rgwConfig *rgwConfig) *apps.Deployment {
```

Comment:
how about renaming this method to `createDeployment()` since it doesn't start it anymore?